exercises/crypto-square/description.md

```diff
@@ -22,10 +22,15 @@ is normalized to:
 "ifmanwasmeanttostayonthegroundgodwouldhavegivenusroots"
 ```
 
-The plaintext should be organized in to a rectangle.  The size of the
-rectangle (`r x c`) should be decided by the length of the message,
-such that `c >= r` and `c - r <= 1`, where `c` is the number of columns
-and `r` is the number of rows.
+The plaintext should be organized in to a rectangle. The size of the
+rectangle should be decided by the length of the message.
+
+If `c` is the number of columns and `r` is the number of rows, then for
+the rectangle `r x c` find the smallest possible integer `c` such that:
+
+- `r x c >= length(message)`,
```

Comment:
What is `x` here?

Reply:
I guss it's a multiplication sign. And if you think that's not necessarily obvious I would agree.

Reply:
Yeah, it's multiplication. It was used as "by" ("r by c") to express the dimensions of the square, and here I used it as multiplication. Should I change both to `*`, or just this one? I remember we wanted to be consistent with the symbol usage thorough the whole repo, but I'm not sure which one is the preferred one.

Reply:
I would either use `*` or `⋅`. This could also be understood as`r times x times c`.

Probably belongs in a separate PR/issue, I don't mean to block this PR because of this.

> I remember we wanted to be consistent with the symbol usage thorough the whole repo, but I'm not sure which one is the preferred one.

I'm not sure either. I think ideally we'd have https://github.com/exercism/website/issues/476 and use proper notation instead of these workarounds.